src/config.js

```diff
@@ -12,29 +14,34 @@ type ApplyConfigToArgvParams = {|
   argv: Object,
   configObject: Object,
   defaultValues: Object,
+  configFileName?: string,
```

Comment:
This should be a required argument. Why did you want it optional? For the tests?